### PR TITLE
BZ2036296: Removed storage class annotations for Azure and GCP

### PR DESCRIPTION
**Pull request by darshan-nagaraj**
The following fixes are included in this PR:

- When cluster is installed, by default storageClass is installed. Removed `is-default-class` annotations in Azure and GCP object definition topics.

- Added cluster support for more than one StorageClass in Storage class annotations topic.

BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2036296

OCP version: 4.6+

Direct Doc Preview Link: 
https://deploy-preview-40925--osdocs.netlify.app/openshift-enterprise/latest/storage/dynamic-provisioning.html#azure-disk-definition_dynamic-provisioning

https://deploy-preview-40925--osdocs.netlify.app/openshift-enterprise/latest/storage/dynamic-provisioning.html#gce-persistentdisk-storage-class_dynamic-provisioning

https://deploy-preview-40925--osdocs.netlify.app/openshift-enterprise/latest/storage/dynamic-provisioning.html#storage-class-annotations_dynamic-provisioning

@duanwei33 PTAL at the PR and let me know your feedback.

### CODE REVIEW

File: modules/dynamic-provisioning-annotations.adoc [hunk 1]
```diff
@@ -28,7 +28,7 @@ metadata:
 
 This enables any persistent volume claim (PVC) that does not specify a
 specific storage class to automatically be provisioned through the
-default storage class.
+default storage class. However, your cluster can have more than one StorageClass, but only one of them can be the default StorageClass.
```
Comment (ahardin-rh) [inline modules/dynamic-provisioning-annotations.adoc:31]: s/StorageClass/storage class 

per guidance in https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references

**Comment by ahardin-rh** [comment]
@darshan-nagaraj Just one comment from me. Thanks!

File: modules/dynamic-provisioning-annotations.adoc (hunk 1 above)
Comment (darshan-nagaraj) [reply modules/dynamic-provisioning-annotations.adoc]: Fixed! Thanks.

**Pull request merged**